peps/pep-0779.rst

```diff
@@ -0,0 +1,174 @@
+PEP: 779
+Title: Criteria for supported status for free-threaded Python
+Author: Thomas Wouters <thomas@python.org>,
+        Matt Page <mpage at python.org>,
+        Sam Gross <colesbury at gmail.com>
+Status: Draft
+Type: Standards Track
+Created: 03-Feb-2025
```

Comment:
This is the date the PEP number was assigned:
```suggestion
Created: 13-Mar-2025
```

Reply:
Fixed in https://github.com/python/peps/pull/4302